### PR TITLE
expand home dir for token file for kuksa viss client

### PR DESCRIPTION
**Pull request by wenwenchenbosch**
solve #207

### CODE REVIEW

File: kuksa_viss_client/__init__.py [hunk 1]
```diff
@@ -63,6 +63,8 @@ def _sendReceiveMsg(self, req, timeout):
     def authorize(self, token=None, timeout = 2):
         if token == None:
             token = self.tokenfile
+        token = os.path.expanduser(token)
+        print("token is " + token)
```
Comment (SebastianSchildt) [inline kuksa_viss_client/__init__.py:67]: Why printing?

Comment (wenwenchenbosch) [reply kuksa_viss_client/__init__.py:67]: maybe better to give a suggestion, so I can use this cool feature from github to fix it more quickly :)
```suggestion
```

**Comment by wenwenchenbosch** [comment]
@SebastianSchildt my github commit using `suggestion` is verified automatically

**Review by SebastianSchildt — APPROVED** [review]
Works 👍

**Pull request merged**